### PR TITLE
Feature/#98/add withdraw button back

### PR DESCRIPTION
**Pull request by alfetopito**
# Description

Part of #98 

Add withdraw button back (was removed during refactoring).

Withdraw and claim are possible again :)

# To Test
1. Make sure there's at least 1 active strategy
2. Request withdraw for one active strategy

- [ ] Tx to request withdraw should be created

3. After signing the tx, change tabs (at the moment required to update the strategies)

- [ ] Withdraw button should now be disabled and show a countdown for given strategy
- [ ] After countdown is over, withdraw button should be enabled with the text `Claim`

4. Claim your funds for given strategy

- [ ] Tx to claim should be created

5. After signing the tx, change tabs (at the moment required to update the strategies)

- [ ] No request/claim button should be displayed for given strategy

# Background

I avoided doing much rework on this withdraw button logic keeping it mostly the same as the previous version https://github.com/gnosis/safe-cmm-app-react/blob/master/src/routes/Strategies/Active.js#L31

There are some caveats with the current implementation, not sure all of them were there before.
Let's follow step by step:

1. Strategy is active, request button is enabled
screenshot_2020-10-23_10-11-24

2. Claim has been requested. Transaction pop-us displayed
screenshot_2020-10-23_10-11-51
  Here, either the user chose to `submit` the transaction or not. We can't tell which from this point forward.
  The button will remain disabled while the screen is active.
  :warning: If the dropdown is closed and re-opened, this state will be lost, and the button will be enabled again

3. If the tx is executed, and the user switches to another tab and back, triggering a refresh on the strategies, the button will show the claim countdown.
screenshot_2020-10-23_10-13-44
  :warning: This state is not triggered automatically because we don't have background strategy update at the moment.

4. Once countdown is over, claim button is enabled
screenshot_2020-10-23_10-19-27

5. If the user had switched to a different screen after the countdown is over, the balances will be reset (notice the amounts at the top on the previous image)
screenshot_2020-10-23_10-19-55
   :warning: Brackets no longer have balances because withdraw has been requested. We might need a different way to query balances, worth checking how `dex-react` does it

6. Claim has been requested. Transaction pop-us displayed
screenshot_2020-10-23_10-20-16
  :warning: Same caveats as step `2` applies. If closing dropdown or switching to a different tab state is reset

7. If the tx is executed, and the user switches to another tab and back, triggering a refresh on the strategies, the button will no longer be displayed
screenshot_2020-10-23_10-20-48
  This is a closed strategy, should not be displayed on the active tab

----

In summary, we need background strategy refresh, and further refactor the component to better handle state changes as well as storing more state on global state.

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 * **🔐 [Safe app link](https://pr131--safecmmappreact.review.gnosisdev.com)**: To be added as a custom app on [Safe Multisig Rinkeby](https://rinkeby.gnosis-safe.io/) or [Mainnet](https://gnosis-safe.io/)
 * **📖 [Storybook](https://pr131--safecmmappreact.review.gnosisdev.com/storybook/)**: Component stories
